sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -158,3 +158,47 @@ directive:
   transform: >
     delete $["x-accessibility"]
 ```
+
+# Add OciMediaType extensible enum
+``` yaml
+directive:
+  from: swagger-document
+  where: $.definitions
+  transform: >
+    $["OciMediaType"] = {
```

Comment:
Why are we adding this type via configuration for .NET SDK only? shouldn't we update the swagger if it's a common type for all the languages?

Reply:
+1 Should Python follow the same settings here?

Reply:
@YalinLi0312, let's discuss this in our next stand-up, whether we want this at all, and if so, if we should put it to a common place (per @ShivangiReja's good comment)